### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by c16u**
Fixed conventions, broken link(s), linked to Wikipedia, and italicized foreign terms.

### Title of change
_why the change and what has changed_

### Checklist

- [ ] Unit tests written
- [ ] Manually tested
- [ ] Prettier & ESLint were run
- [ ] New dependencies are included in `package-lock.json`

### Screenshot
_insert screenshot if needed_

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32858

File: src/containers/About/index.js [hunk 1]
```diff
@@ -11,11 +11,10 @@ export default () => (
       <div className="row">
         <div className="col-md-8 col-md-offset-2">
           <h4 className="source-sans">
-            The Noble Qur’an is the central religious text of Islam. Muslims
-            believe the Qur’an is the book of Divine guidance and direction for
+            The Noble Qur’an is the central religious text of <a href="https://en.wikipedia.org/wiki/Islam">Islam</a>.
```
Comment (naveed-ahmad) [inline src/containers/About/index.js]: Maybe its time to add more static pages within application, wikipedia isn't reliable source. 
https://en.wikipedia.org/wiki/Islam#Criticism

**Comment by mmahalwy** [comment]
please hold off on merging this!

File: src/containers/About/index.js (hunk 1 above)
Comment (c16u) [reply src/containers/About/index.js]: @naveed-ahmad 
https://en.m.wikipedia.org/wiki/Wikipedia:Neutral_point_of_view

Comment (naveed-ahmad) [reply src/containers/About/index.js]: @c16u Maybe we want "true point of view" ? :)

**Comment by naveed-ahmad** [comment]
@c16u Sorry, had to close this PR. I'm not sure why we need to add Christians, and Jews criticism on OUR Prophet( PBUH ). 

What we should do is add more static pages about Islam, Quran, Muhammad(PBUH) etc. We can took the content from **The Clear Quran** translation to get started with.

**Comment by c16u** [comment]
@naveed-ahmad Where did you get the idea that Christians and Jews wrote that section? Other religions have a criticism section as well.
https://en.wikipedia.org/w/index.php?title=Christianity&mobileaction=toggle_view_desktop#Criticism_and_apologetics

In addition, I made way more changes than link to Wikipedia, so why not accept the remaining changes? By the way, which a link already existed—i.e. link to the Qur'an article. You can't have your cake and eat it—either remove all the links to Wikipedia (in the intro mainly) or accept all my changes. :) After all, Islam is not such a weak religion that a bit of critiquing will cause it to collapse.

**Comment by naveed-ahmad** [comment]
@c16u Salam brother, sorry if was unable to explain properly. At one point, we were showing surah info from wikipedia. Then this was replaced with other source: https://github.com/quran/quran.com-frontend/issues/82, https://github.com/quran/quran.com-frontend/issues/215

Reason is same, wikipedia is not credible source. Here is what Sheikh Yusuf Estes has to say about wikipedia https://youtu.be/lqyQg2eBlJc?t=6m25s

I guess only link we've to wikipedia now is [Quran](https://en.wikipedia.org/wiki/Quran) and we should replaced that as well. 


**Where did you get the idea that Christians and Jews wrote that section**

> Early criticism came from Christian authors, many of whom viewed Islam as a Christian heresy or a form of idolatry and often explained it in apocalyptic terms.[348] Later there appeared criticism from the Muslim world itself, and also from Jewish writers and from ecclesiastical Christians.[349][350][351]

> Objects of criticism include the morality of the life of Muhammad, the last law bearing prophet of Islam, both in his public and personal life

And source they've listed was from Catholic Encyclopedia.. 

Hopefully, we're on same page now. That link isn't adding any value to quran.com, we should add more static pages for Quran, Islam and other terms. That'll also help improving seo of site plus we'll have content from authentic source.

**Pull request closed without merging**